MessengerOnFileForLinux/MessageFramework/test/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# TEST FILES WITH GTEST/GMOCK UNIT TESTS
+set(TEST_FILES ${TEST_FILES}
+    ${CMAKE_CURRENT_SOURCE_DIR}/StringSumTest.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/MessageTest.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/MessageTestCore.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/MessageTest.cpp PARENT_SCOPE)
+
+#SOURCES FILES WHICH ARE NEEDED TO COMPILE TESTS
```

Comment:
Do wyrzucenia, to jest tylko potrzebne (nie wiem czemu) tylko w tym jednym CMakeu z którego to kopiowałes

Reply:
Tak, ale tprzyszłościow chcielibyśmy by CMake dobierał tylko te mu potrzebne, więc w każdym pliku będzie pojedyńcze dodanie tego. W takim razie uważam, żeby zostawić

Reply:
Wyrzuć to, aktualnie nam to nie potrzebne a jest tylko śmietnik w tmy CMaku